docs/product-getting-started-guide/src/main/asciidoc/decision-table-create-proc.adoc

```diff
@@ -0,0 +1,107 @@
+[id='dec-table-create-proc']
```

Comment:
Wrong concept module before this one. After all our back-and-forth about Decision tables file, it's actually the wrong choice anyway. Should be the Guided decision table concept file (guided-decision-tables-con.adoc) not the Decision tables concept. They are for different tools.

Reply:
Well, that make more sense :) Done